splink/duckdb/duckdb_comparison_library.py

```diff
@@ -36,48 +36,58 @@ def _null_level(self):
     def _else_level(self):
         return else_level
 
+    @property
+    def _datediff_level(self):
+        return datediff_level
+
+    @property
+    def _array_intersect_level(self):
+        return array_intersect_level
+
```

Comment:
by including these here then any new library comparison can access both of these levels straightaway, rather than having to repeatedly add these properties to any comparison that wants to make use of them - seems more consistent to have them all accessible here.